dol/src/lib.rs

```diff
@@ -24,6 +24,8 @@ pub enum Error {
     OverlappingSections(u32, u32),
     #[error("Section sizes too large")]
     SectionsTooLarge,
+    #[error("Out of Bound: {0:08X}")]
+    OutOfBound(u32)
```

Comment:
```suggestion
    OutOfBounds(u32)
```
Maybe let's call the error something more descriptive like `attempted to access {0:08X} past DOL bounds`?